newm-server/src/main/kotlin/io/newm/server/features/release/model/Items.kt

```diff
@@ -0,0 +1,9 @@
+package io.newm.server.features.release.model
+
+import com.google.gson.annotations.SerializedName
+
+data class Items(
+    @SerializedName("id") var id: String,
+    @SerializedName("track_number") var trackNumber: Int,
+    @SerializedName("uri") var uri: String
+)
```

Comment:
Why vars instead of vals here?

Reply:
I would ask the same question, as per my understanding `vals` are read only; val cannot be reassigned just like final, right? attributes will be mapped to the response coming back from spotify api, so not sure if that will be considered `reassign-ing` ?!

Reply:
If they'

> I would ask the same question, as per my understanding `vals` are read only; val cannot be reassigned just like final, right? attributes will be mapped to the response coming back from spotify api, so not sure if that will be considered `reassign-ing` ?!

If they don't need to be modified anywhere ion our code after after network deserialization, change all to `val`.